### DISCUSSION TITLE
Where is DirectX 12 in Heroic Launcher for FortniteGame?

### DISCUSSION THREAD

**Discussion by Richeduserus** [community/community#106401] [Programming Help]
### Body

Welcome,

I'm having trouble finding the option to enable DirectX 12 in Heroic Launcher for Fortnite (Windows 11). Does anyone know where I can locate this setting? I know that DirectX 12 can significantly improve my gaming performance and I wanted to give it a try. Any help would be appreciated!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
